site/journal/Ember Template Imports/Part 1.md

```diff
@@ -319,7 +319,7 @@ A `<template>` tag contains the template content, which will be *compiled* to an
         ```js
         export const Greet = <template>Hello!</template>
 
-        const Conditional = hbs`{{#if @val}}, {{@val}}`;
+        const Conditional = hbs`{{#if @val}}, {{@val}}{{/if}}`;
```

Comment:
Wasn't sure about this - is this supposed to be within a template tag too?

Reply:
Yep, you were absolutely right. 😅